app/editor/src/features/content/form/components/tags/Tags.tsx

```diff
@@ -1,28 +1,48 @@
 import { IContentForm } from 'features/content/form/interfaces';
 import { useFormikContext } from 'formik';
+import _ from 'lodash';
 import React from 'react';
 import { FaListAlt } from 'react-icons/fa';
 import { useLookup } from 'store/hooks';
-import { Button, Col, FieldSize, IOptionItem, ITagModel, Row, Select } from 'tno-core';
+import { Button, Col, FieldSize, IOptionItem, Row, Select } from 'tno-core';
 
 import { DraggableTagList } from './DraggableTagList';
 import * as styled from './styled';
 
 export interface ITagsProps {
-  selectedExternal?: string[];
+  defaultTags?: string[];
 }
 
 /**
  * The component that renders tags for a given text field
  * @returns the Tags component
  */
-export const Tags: React.FC<ITagsProps> = ({ selectedExternal }) => {
+export const Tags: React.FC<ITagsProps> = ({ defaultTags }) => {
   const { values, setFieldValue } = useFormikContext<IContentForm>();
   const [{ tags }] = useLookup();
+
   const [showList, setShowList] = React.useState(false);
-  const [initialTagsLoad, setInitialTagsLoad] = React.useState(true);
-  const [externalTags, setExternalTags] = React.useState<ITagModel[]>([]);
-  const [manualTags, setManualTags] = React.useState<ITagModel[]>([]);
+  const [tagOptions, setTagOptions] = React.useState(
+    tags.map((tag) => {
+      return {
+        label: tag.code,
+        value: tag.id,
+        isDisabled: !tag.isEnabled,
+      } as IOptionItem;
+    }),
+  );
+
+  React.useEffect(() => {
+    setTagOptions(
```

Comment:
should we just filter tags that are not enabled here?  This list could get very large

Reply:
Done